src/styles/blocks/card.scss

```
@@ -0,0 +1,132 @@
+$color-navy-blue: #060b35;
+$color-light-gray: #616070;
+$color-blue: #00acdc;
+$card-width: 200px;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: #{$card-width};
```

```suggestion
  width: $card-width;
```
Is this syntax you have something else than a variable? Cause for a variable it's less code to use $variable-name.

I believe this syntax is to reference the variable you declared in a for loop. But for the variables you declared in your variables.scss you can use this syntax.